src/App.js

```diff
@@ -1,9 +1,71 @@
-function App () {
+
+import {BirdCards} from "./Components/BirdCards";
+import birdData from "./data/birds"
+import { useState } from "react";
+import { Cart } from "./Components/Cart";
+import { Checkout } from "./Components/Checkout";
+import './App.css'
+
+function App() {
+  const [birds, setBirds]= useState([]);
+
+  // MUTAR consiste en CAMBIAR El contenido INTERNO de una estructura de datos (ARRAY / OBJETOS)
+  const adopt1 = (newBird) => {
+    const randomExtraId = Math.round(Math.random() * 1000) // 0,685943674356124 ==> 4444-5785
+    
+    let actualObject = {
+      id: `${newBird.id}${randomExtraId}`, /// 44441753
```

Comment:
Nice use of a more complex ID here to deal with multiples. Could also get around that by accessing the bird by index/position in the cart, but this way seems cleaner.